homeTask_3/homeTask_3/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace homeTask_3
+{
+    class Program
+    {
+
+        //Задание- создать три массива по 10 элементов, первый
+        //заполнить значениями индексов элементов, второй - квадратами индексов,
+        //третий-кубами, и вывести на экран средние значения элементов для всех массивов
+
+        static void InitializeArray(int[] array, int n)
```

Comment:
why you use static?

Reply:
why can't I use static methods. In this problem, I can create three arrays and use static methods at the class level instead of at the object level. Maybe I don't understand something, please correct me. In general, at this stage, we did not go into details and at the training, in the examples, we always used only static methods.

Reply:
Because it will kill you in future =)
Of course you can use static, but why you use it? ask yourself this question.
Read this [link](https://habr.com/ru/post/206082/)
In simple console app it works fine, but in future it will make alot of trash. In your case it will make alot of 'fun' in testing.
In my opinion we should learn how to write good code from start. It save your time in future.

Reply:
@EgorAntonovich I've fixed it. Could you please review it?